From aa104a32d52e686f9e9fcc8a9ca3978f50f4ab7b Mon Sep 17 00:00:00 2001
From: Konstantin Tokarev <annulen@yandex.ru>
Date: Wed, 3 Jun 2020 15:01:42 +0300
Subject: [PATCH 03/19] Fix compilation with Python 3.9: avoid passing encoding
 to json.load()

In Python 2.7 UTF-8 is assumed by default, while in Python 3 this argument
is not supported.

Change-Id: Ic459d60a6b20bc1838d8771bc36ac41614fe61a9
---
 qtwebkit/Source/JavaScriptCore/generate-bytecode-files | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qtwebkit/Source/JavaScriptCore/generate-bytecode-files b/qtwebkit/Source/JavaScriptCore/generate-bytecode-files
index c5dab429c7..af3431275e 100644
--- a/qtwebkit/Source/JavaScriptCore/generate-bytecode-files
+++ b/qtwebkit/Source/JavaScriptCore/generate-bytecode-files
@@ -163,7 +163,7 @@ if __name__ == "__main__":
         initBytecodesFile = openOrExit(initASMFileName, "w")
 
     try:
-        bytecodeSections = json.load(bytecodeFile, encoding = "utf-8")
+        bytecodeSections = json.load(bytecodeFile)
     except:
         print("Unexpected error parsing {0}: {1}".format(bytecodeJSONFile, sys.exc_info()))
 
-- 
2.43.0

